clang_tools/main.py

```diff
@@ -18,7 +20,9 @@ def get_parser() -> argparse.ArgumentParser:
         "-i",
         "--install",
         metavar="VERSION",
-        help="Install clang-tools about a specific version.",
+        help="Install clang-tools about a specific version. This can be in the form of"
+        " a semantic version specification (``x.y.z``, ``x.y``, ``x``) or a path that "
+        "points to the directory where the binaries already exist.",
```

Comment:
oh, so just remove the prompt? That's reasonable.

Reply:
As for changing the `--help`, maybe we just mention that a malformed version silently fails (and remove the `assert path.exists()`).

Reply:
The whole point of this PR is because cpp-linter also takes a path as the version input (in case the clang tools were installed by some other means). Therefore, we can't fail if the version is not a semantic specification; we can only `sys.exit(0)`.

I think I'm misunderstanding something here.

Reply:
Just so I understand, do you prefer to have the prompt about "non-semantic version specified"?

Reply:
I could easily drop the commit that removed the prompt.

Reply:
ok I dropped that commit.